felix/bpf/bpf.go

```diff
@@ -2230,6 +2233,13 @@ func SupportsBPFDataplane() error {
 	return nil
 }
 
+func SupportIPIPAsL3Device() bool {
```

Comment:
It does not "support" it either is or is not, right?

```suggestion
func IPIPDeviceIsL3() bool {
```